opm/material/fluidmatrixinteractions/EclMaterialLawManager.hpp

```diff
@@ -117,8 +117,7 @@ class EclMaterialLawManager
                       const std::vector<int>& compressedToCartesianElemIdx)
     {
         // get the number of saturation regions and the number of cells in the deck
-        int ntsFun = deck.getKeyword("TABDIMS").getRecord(0).getItem("NTSFUN").get<int>(0);
-        unsigned numSatRegions = static_cast<unsigned>(ntsFun);
+        const size_t numSatRegions = eclState.runspec().tabdims().getNumSatTables();
```

Comment:
Just generally trying to move towards a less noisy interface.

If you look at the `EclipseConfig` (the most recent addition really) it does this. The `get`-prefix'd still exist mostly for compatibility at this point (in my not so humble opinion), and I'd like to both clean up headers, restrict includes and preserve version the parser interfaces in the not-so-distant future.